tests/test_modeling_common.py

```diff
@@ -746,6 +746,19 @@ def _create_and_check_torchscript(self, config, inputs_dict):
                     traced_model = torch.jit.trace(
                         model, (input_ids, bbox, image), check_trace=False
                     )  # when traced model is checked, an error is produced due to name mangling
+                elif config.__class__.__name__ == "GraphormerConfig":  # Graphormer requires additional inputs
```

Comment:
`Graphormer` needs more inputs.

Reply:
Really not in favor of changing this test just for Graphormer. Please overwrite the test in the Gtaphormer test file.